Pipfile

```diff
@@ -12,4 +12,4 @@ pytest = "*"
 [packages]
 
 [requires]
-python_version = "3.7"
+python_version = "3.8"
```

Comment:
Would like to confirm that this okay, as we currently say that we support all the way back to 3.6.

Reply:
This is ok. Should also update the docs

Reply:
It breaks the test build, so it's going.